extensions/amp-analytics/0.1/instrumentation.js

```diff
@@ -484,10 +484,17 @@ export class InstrumentationService {
    * @private
    */
   createTimerListener_(listener, timerSpec) {
-    const intervalId = this.win_.setInterval(
-        listener.bind(null, new AnalyticsEvent(AnalyticsEventType.TIMER)),
-        timerSpec['interval'] * 1000);
-    listener(new AnalyticsEvent(AnalyticsEventType.TIMER));
+    const
```

Comment:
apologies. I had meant to say this earlier but might have forgotten. It would be preferable to initialize each variable in its own statement. Something like:

```
    const hasImmediateProp = timerSpec.hasOwnProperty('immediate'),
    const callImmediate = hasImmediateProp ? Boolean(timerSpec.immediate) : true,
    const intervalId = this.win_.setInterval(
           listener.bind(null, new AnalyticsEvent(AnalyticsEventType.TIMER)),
           timerSpec['interval'] * 1000);
 ```

It is more readable this way and the compiler will do the optimization for us.